src/games/strategy/triplea/ui/menubar/ExportMenu.java

```diff
@@ -112,7 +112,7 @@ private void exportXMLFile() {
     final String xmlFile;
     try {
       gameData.acquireReadLock();
-      final GameDataExporter exporter = new games.strategy.engine.data.export.GameDataExporter(gameData, false);
+      final GameDataExporter exporter = new games.strategy.engine.data.export.GameDataExporter(gameData);
```

Comment:
Would you mind adding an import statement instead of writing the full class name? Otherwhise no objections on merging...